database/migrations/2019_11_14_203229_add_pending_count_to_campaigns.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddPendingCountToCampaigns extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->integer('pending_count')->default(0);
+            $table->integer('accepted_count')->default(0);
```

Comment:
Can we put these columns after `end_date` so that the timestamps are still the last columns on this table?

Reply:
Oh sure!